client-participation/index.html

```diff
@@ -10,6 +10,8 @@
     <link rel="stylesheet" type="text/css" href="<%= basepath %>/css/polis.css"> <!-- proxy all CSS through server for respond.js -->
 
     <script>
+      window.userObject = {};
```

Comment:
Is this needed to be initialised because failing elsewhere?

Reply:
Yes this is referenced a fair bit client side but is never assigned from a clean state - it is assigned to a copy of itself in a Backbone class but this also crashes if the variable isn't initialised